infra/ursa/docker-compose.yml

```diff
@@ -75,13 +75,12 @@ services:
       dockerfile: Dockerfile
     restart: on-failure
     ports:
-      # 6009 TCP and UDP, used by the P2P protocol running the network
       - "6009:6009"
+      - "4069:4069"
```

Comment:
4069 doesn’t need to be exposed, as we use nginx in front

Reply:
Hmm this actually opens an interesting problem: nodes currently register with the tracker with their raw internal port, there's no consideration for the proxy or https atm. I think this works temp until #151 is completed, which would make the necessary changes here. Also, the prometheus container is scraping ursa:4069, so it needs to be exposed for that internally.

Reply:
We can draft then until that's ready

Reply:
Nodes should register their domain ie tls

Reply:
Using ports exposes on the Host you can scrape with no problem

Reply:
> Using ports exposes on the Host you can scrape with no problem

Ah this is the missing link, perfect. So I can just remove the ports and leave expose

Reply:
https://github.com/fleek-network/ursa/pull/163/commits/283935721376edbf60cbb18478de76f7325c19b8